resources/views/bread/edit-add.blade.php

```diff
@@ -183,7 +183,7 @@ function deleteHandler(tag, isMulti) {
             $('.form-group').on('click', '.remove-single-file', deleteHandler('a', false));
 
             $('#confirm_delete').on('click', function(){
-                $.post('{{ route('voyager.media.remove') }}', params, function (response) {
+                $.post('{{ route('voyager.'.$dataType->slug.'.remove') }}', params, function (response) {
```

Comment:
Route name reads like you're removing the `$slug`, not media.  Please rename routes per my previous comment (`voyager.[$slug].media.remove`, or similar)